src/components/templates/Playa/Playa.tsx

```diff
@@ -803,7 +804,7 @@ const Playa = () => {
         movingLeft={movingLeft}
         movingRight={movingRight}
         setMyLocation={setMyLocation}
-        setSelectedUserProfile={setSelectedUserProfile}
+        setSelectedUserProfile={setUserProfile}
```

Comment:
Technically we should probably just remove this prop entirely + use the hook within this component. but `Playa` is a legacy template that is in the process of being deleted, and `AvatarLayer` seems to only be called by it, so no need to worry about that here.